live-examples/html-examples/text-content/meta.json

```diff
@@ -18,6 +18,16 @@
             "title": "HTML Demo: <dl>",
             "type": "tabbed"
         },
+        "dd": {
+            "baseTmpl": "tmpl/live-tabbed-tmpl.html",
+            "exampleCode":
+                "live-examples/html-examples/text-content/dd.html",
+            "cssExampleSrc":
+                "live-examples/html-examples/text-content/css/dd.css",
+            "fileName": "dd.html",
+            "title": "HTML Demo: <dd>",
+            "type": "tabbed"
+        },
```

Comment:
Please keep this file in alpha order.

Reply:
I see no reason why this should be confusing; yeah, it has info about HTML in it, but it's also correct, so it's not a problem per se, and being in the box should make it pretty clear that it's an example.